CONSTANTS.py

```diff
@@ -72,8 +72,10 @@ def dumps(cls):
 
 DISPLAY=pyglet.canvas.get_display()#It took ages to find these functions, so don't question them.
 SCREEN=DISPLAY.get_default_screen()
-WIDTH=SCREEN.width
-HEIGHT=SCREEN.height
+SCREEN_MODES=SCREEN.get_modes()
+SCREEN_MODE=SCREEN_MODES[-1]
```

Comment:
on my laptop, the first one is actually the correct one and the last one the smallest one. If this is reversed on windows, I suggest going through all of them and getting the biggest by total pixels.
```
SCREEN_MODES.sort(reversed=True,key=lambda mode:mode.width*mode.height)
SCREEN_MODE=SCREEN_MODES[0]
```

Reply:
Changed with the next commit